indy_client/test/cli/test_nym.py

```diff
@@ -361,3 +364,18 @@ def test_send_different_nyms_succeeds_when_batched(
 
     do('send GET_NYM dest={dest}',
         mapper=parameters, expect=CURRENT_VERKEY_FOR_NYM, within=2)
+
+
+def test_nym_resend(looper, sdk_pool_handle, sdk_wallet_steward):
```

Comment:
This test should not be in the CLI folder

Reply:
Ok, moved to separate file.